docker/mint/Dockerfile

```diff
@@ -1,6 +1,4 @@
-FROM debian:buster-20200803@sha256:a44ab0cca6cd9411032d180bc396f19bc98f71972d2398d50460145cab81c5ab AS debian-base
-
-FROM debian-base AS toolchain
+FROM debian:buster-20200803@sha256:a44ab0cca6cd9411032d180bc396f19bc98f71972d2398d50460145cab81c5ab AS toolchain
```

Comment:
doing this to make the first 20 or so lines of docker files identical across files.   Thought my change might have been adding time but it was a change to the validator docker file.   Still want the lines identical so I can add a test.

Reply:
I'd suggest move the mint's Dockerfile change to a separate commit or PR.